docs/customizations/custom-slots.md

```
@@ -5,13 +5,28 @@ outline: false
+- `security`
+- `parameters`
+- `request-body`
+- `responses`
+- `playground`
+- `code-samples`
+- `branding`
+- `footer`
+
+In the following example, we demonstrate how to use all available slots in the `OAOperation` component. Each slot is populated with a `SlotDebugger` component that displays the slot properties for debugging purposes.
+
+<ExampleBlock :code-section-title="null" :example-section-title="null">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: ExampleBlock

(MD033, no-inline-html)